rep-I0001.rst

```diff
@@ -109,9 +108,11 @@ The dynamic joint state is meant to mimic both the ROS JointState and FollowJoin
         accelerations[] 
         effort[]
         position_desired[]
-        position_errors[]
+        position_error[]
         velocity_desired[]
-        velocity_errors[]
+        velocity_error[]
+        accel_desired[]
+        accel_error[]
```

Comment:
This message maps to both the [JointState](http://docs.ros.org/api/sensor_msgs/html/msg/JointState.html) and [FollowJointTrajectoryFeedback](http://docs.ros.org/api/control_msgs/html/action/FollowJointTrajectory.html), which eventually encapsulates data using a [JointTrajectoryPoint](http://docs.ros.org/api/trajectory_msgs/html/msg/JointTrajectoryPoint.html).  These messages use different conventions, so there is no "best choice".  In the end, I updated `error` to match `FollowJointTrajectoryFeedback`, but left the pos/vel/accel naming as it was in the original REP.  I'm happy to use something more consistent, if it makes sense.

For what it's worth, the `DynamicJointPoint` maps only to `JointTrajectoryPoint`, so using "mostly plural" (`effort` is singular) to match the ROS message makes sense there.  I could buy an argument that we should match everything to `JointTrajectoryPoint`'s naming, for consistency.